data/mods/TEST_DATA/known_bad_density.json

```diff
@@ -622,6 +622,7 @@
       "foodplace_food",
       "brew_dandelion_wine",
       "badge_deputy",
+      "art_badge",
```

Comment:
You're not supposed to add new things to this list. If an item fails density checks, you need to fix it by adjusting its weight, volume or possibly material.

Reply:
I fixed the offending badge_abstract base item and all its children, and removed them from known bad densities.